### PR TITLE
add 3.2 version, compatibility table updates

### PR DESCRIPTION
**Pull request by omegabytes**
### Description

* Adds 3.2 to supported versions
* Updates compatibility table
* Fixes an existing malformed url

Checked for: 
* Trailing ticks/commas
* Missing ticks around field/entity/resource names
* Broken links
* Casing and spacing in url tags
* Max length for description and resolution at 80 chars (does not include indents)

### Testing instructions

Netlify link: https://deploy-preview-5163--kongdocs.netlify.app/konnect/runtime-manager/troubleshoot/

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/_data/tables/plugin_index.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@
       enterprise: Yes
       konnect: Yes
       network_config_opts: All
-      notes: --
+      notes: Disabling the hmac-sha1 algorithm when using the FIPS image is not supported in konnect.
```
Comment (lena-larionova) [inline app/_data/tables/plugin_index.yml:28]:
```suggestion
      notes: |
        Disabling the `hmac-sha1` algorithm when using the FIPS image is not supported in Konnect.
```

File: app/_data/tables/version_errors_konnect.yml [hunk 2]
```diff
@@ -255,6 +255,472 @@ messages:
...
+    DocumentationURL: |
+      [Request Transformer Advanced plugin](/hub/kong-inc/request-transformer-advanced/)
+  - ID: D155
+    Severity: error
+    Description: |
+      Plugin `datadog-tracing` is not available in Kong Gateway versions < 3.2.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.2` or above.
+    DocumentationURL: |
+      [Datadog Tracing plugin](/hub/kong-inc/datadog-tracing/)
```
Comment (omegabytes) [inline app/_data/tables/version_errors_konnect.yml:546]: This endpoint doesn't exist. Where should it point?

Comment (lena-larionova) [reply app/_data/tables/version_errors_konnect.yml:546]: It doesn't exist yet, but will once the 3.2 doc is published. This is correct.

File: app/_data/tables/version_errors_konnect.yml [hunk 3]
```diff
@@ -255,6 +255,472 @@ messages:
...
+    Description: |
+      For the `service` entity, one of `tls_verify`, `tls_verify_depth`,
+      `ca_certificates` or `client_certificate` fields has been set with `protocol`
+      set to `TLS`. This is not supported in Kong Gateway versions < 3.2.0.0. These
+      fields have been removed from the service configuration in the data-plane and
+      the service has been disabled.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.2.0.0` or above.
+    DocumentationURL: |
+      [Service entity](/gateway/latest/admin-api/#service-object)
```
Comment (mikefero) [inline app/_data/tables/version_errors_konnect.yml:723]: Missing D170

```
{
  "ID": "D170",
  "Severity": "error",
  "Description": "For the 'rate-limiting-advanced' plugin, 'config.strategy' field has been set to 'cluster'. This is not supported in Kong Gateway versions < 3.2.0.0. The plugin configuration has been changed to 'config.strategy=local' and 'sync_rate=null' in the data-plane and might not work as intended.",
  "Resolution": "Please upgrade Kong Gateway to version '3.2.0.0' or above.",
  "DocumentationURL": ""
}
```

File: app/_data/tables/version_errors_konnect.yml [hunk 4]
```diff
@@ -499,3 +965,390 @@ messages:
...
+    Severity: error
+    Description: |
+      For the `aws-lambda` plugin, one or more of the following `config` fields are
+      set: `aws_imds_protocol_version` but Kong Gateway versions < 3.2 do not support
+      these fields. Plugin features that rely on these fields are not working as
+      intended.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.2` or above.
+    DocumentationURL: |
+      [AWS Lambda plugin](/hub/kong-inc/aws-lambda/)
```
Comment (mikefero) [inline app/_data/tables/version_errors_konnect.yml:1354]: Missing P162

```
{
  "ID": "P162",
  "Severity": "error",
  "Description": "For the 'opentelemetry' plugin, instances have been configured on a specific service, route, or consumer. This is not supported in Kong Gateway versions < 3.2. The plugin has been removed in the data-plane.",
  "Resolution": "Unset the 'service', 'route', and 'consumer' fields for the plugin, or upgrade Kong Gateway to a version >= 3.2.",
  "DocumentationURL": ""
}

```

**Review by mikefero — CHANGES_REQUESTED** [review]
Looks great @omegabytes, thanks for updating these. I noticed `D170` and `P162` were missing from the list. I put the generated JSON near the items in the list.

Kong Gateway 3.2.0.0 has been released, this should be the last item needed before the update.

**Comment by lena-larionova** [comment]
We had to merge 3.2 already, as the release went out yesterday. 
I've left the 3.2 branch up so that this PR doesn't need heavy rebasing; we'll cherry-pick it to main once it's merged.

File: app/_data/tables/version_errors_konnect.yml (hunk 4 above)
Comment (omegabytes) [reply app/_data/tables/version_errors_konnect.yml]: [166a6d2](https://github.com/Kong/docs.konghq.com/pull/5163/commits/166a6d2ebe1318fc9e0586d487670882a3ac5d12)

File: app/_data/tables/version_errors_konnect.yml (hunk 3 above)
Comment (omegabytes) [reply app/_data/tables/version_errors_konnect.yml:723]: [166a6d2](https://github.com/Kong/docs.konghq.com/pull/5163/commits/166a6d2ebe1318fc9e0586d487670882a3ac5d12)

**Comment by omegabytes** [comment]
`D171` was also missing [5f7de14](https://github.com/Kong/docs.konghq.com/pull/5163/commits/5f7de1452ba80029f5562e97de121892902333c1)

This should be good to go @mikefero

**Comment by mikefero** [comment]
> `D171` was also missing [5f7de14](https://github.com/Kong/docs.konghq.com/pull/5163/commits/5f7de1452ba80029f5562e97de121892902333c1)
> 
> This should be good to go @mikefero

Nice catch!!!

File: app/_data/tables/version_errors_konnect.yml [hunk 5]
```diff
@@ -1351,4 +1362,15 @@ messages:
     Resolution: |
       Please upgrade Kong Gateway to version `3.2` or above.
     DocumentationURL: |
-      [AWS Lambda plugin](/hub/kong-inc/aws-lambda/)
\ No newline at end of file
+      [AWS Lambda plugin](/hub/kong-inc/aws-lambda/)
+  - ID: P162
```
Comment (mikefero) [inline app/_data/tables/version_errors_konnect.yml:1378]: @lena-larionova Does order matter for the rendering of the documentation?

Comment (lena-larionova) [reply app/_data/tables/version_errors_konnect.yml:1378]: It just gets generated in the same order as what's in this file. See:
https://deploy-preview-5163--kongdocs.netlify.app/konnect/runtime-manager/troubleshoot/

Comment (mikefero) [reply app/_data/tables/version_errors_konnect.yml:1378]: Ok thank you, sorry about not looking at the rendered components.

I read the PR wrong anyway so this helped a ton.

**Review by mikefero — APPROVED** [review]

File: app/_data/tables/version_errors_konnect.yml [hunk 6]
```diff
@@ -499,3 +988,401 @@ messages:
...
+      [Keys entity](/gateway/latest/admin-api/#keys-entity)
+  - ID: P142
+    Severity: error
+    Description: |
+      The `key-set` entity is being used, but Kong Gateway versions < 3.1 do not
+      support this entity.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.1` or above.
+    DocumentationURL: |
+      [Key Sets entity](/gateway/latest/admin-api/#key-sets-entity)
```
Comment (lena-larionova) [inline app/_data/tables/version_errors_konnect.yml:1187]:
```suggestion
      [Key Sets entity](/gateway/latest/admin-api/#keys-object)
```
We don't seem to have a separate section for key sets, they're just part of keys.

File: app/_data/tables/version_errors_konnect.yml [hunk 7]
```diff
@@ -499,3 +988,401 @@ messages:
...
+      [AWS Lambda plugin](/hub/kong-inc/aws-lambda/)
+  - ID: P141
+    Severity: error
+    Description: |
+      The `key` entity is being used, but Kong Gateway versions < 3.1 do not support
+      this entity.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.1` or above.
+    DocumentationURL: |
+      [Keys entity](/gateway/latest/admin-api/#keys-entity)
```
Comment (lena-larionova) [inline app/_data/tables/version_errors_konnect.yml:1178]:
```suggestion
      [Keys entity](/gateway/latest/admin-api/#keys-object)
```

File: app/_data/tables/version_errors_konnect.yml [hunk 8]
```diff
@@ -499,3 +988,401 @@ messages:
...
+      [Tags](/gateway/latest/admin-api/#tags)
+  - ID: P135
+    Severity: error
+    Description: |
+      The `vaults` entity is being used, but Kong Gateway versions < 3.1 do not support
+      this entity.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.1` or above.
+    DocumentationURL: |
+      [Vaults entity](/gateway/latest/admin-api/#vaults-entity)
```
Comment (lena-larionova) [inline app/_data/tables/version_errors_konnect.yml:1113]:
```suggestion
      [Vaults entity](/gateway/latest/admin-api/#vaults-object)
```

**Review by lena-larionova — COMMENTED** [review]
minor edits for links

File: app/_data/tables/version_errors_konnect.yml [hunk 9]
```diff
@@ -255,6 +255,495 @@ messages:
...
+  - ID: D146
+    Severity: error
+    Description: |
+      For the `vaults` entity, one or more of the following schema fields are set:
+      `auth_method`, `kube_role`, `kube_api_token_file` but Kong Gateway versions <
+      3.1.0.0 do not support these fields.
+    Resolution: |
+      Please upgrade Kong Gateway to version `3.1.0.0` or above.
+    DocumentationURL: |
+      [Vaults entity](/gateway/latest/admin-api/#vaults-entity)
```
Comment (lena-larionova) [inline app/_data/tables/version_errors_konnect.yml:451]:
```suggestion
      [Vaults entity](/gateway/latest/admin-api/#vaults-object)
```
Missed one.

**Review by lena-larionova — APPROVED** [review]
LGTM! Thanks for putting this whole thing together!

**Pull request merged**